pkg/providers/vsphere/client/client_test.go

```diff
@@ -98,7 +100,7 @@ var _ = Describe("Client", Label(testlabels.VCSim), Ordered /* Avoided race for
 				},
 				CAFilePath:            serverCertFile,
 				InsecureSkipTLSVerify: false,
-				Datacenter:            simulator.Map.Any("Datacenter").Reference().Value,
+				Datacenter:            datacenter.Reference().Value,
```

Comment:
Closest GH will let me comment - a little before here we have `Username: expectedPassword` instead of `expectedUsername` (although both are set to the same value)

Reply:
Will address in a separate fix, thanks!